Directory.Build.props

```diff
@@ -0,0 +1,12 @@
+<Project>
+
+  <PropertyGroup>
+    <TargetFrameworkNETStandard>netstandard2.0</TargetFrameworkNETStandard>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <BuildOutputDirectory>$(MSBuildThisFileDirectory)bin\$(Configuration)\</BuildOutputDirectory>
+    <SupplementalBuildOutputDirectory>$(MSBuildThisFileDirectory)bin\Build$(Configuration)\</SupplementalBuildOutputDirectory>
+    <TestOutputDirectory>$(MSBuildThisFileDirectory)bin\Test$(Configuration)\</TestOutputDirectory>
+    <SupportingBuildTasksDirectory>$(SupplementalBuildOutputDirectory)$(TargetFrameworkNETStandard)\</SupportingBuildTasksDirectory>
```

Comment:
Comment on naming: looks like two different adjectives are used here: Supplemental vs Supporting

Could these be `$(BootstrapOutputDirectory)` and `$(BootstrapTasksDirectory)` to match what we already have for `*.BootstrapTasks.csproj`?

It might also make sense to have more properties for the two assemblies:
```xml
<PrepTasksAssembly>$(BootstrapTasksDirectory)xa-prep-tasks.dll</PrepTasksAssembly>
<BootstrapTasksAssembly>$(BootstrapTasksDirectory)Xamarin.Android.Tools.BootstrapTasks.dll</BootstrapTasksAssembly>
```

Reply:
Good suggestion, I definitely like this more. Seems best to surface properties for the two task assemblies rather than their directory. I may keep `SupplementalBuildOutputDirectory` as named and just remove the `SupportingBuildTasksDirectory` property outright in favor of the two assembly props mentioned above if that sounds good?

Reply:
Sounds good 👍